api/repositories/buildpack_repository_test.go

```diff
@@ -78,7 +78,29 @@ var _ = Describe("BuildpackRepository", func() {
 
 			It("errors", func() {
 				_, err := buildpackRepo.ListBuildpacks(context.Background(), authInfo)
-				Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("no BuilderInfo %q resource found in %q namespace", builderName, rootNamespace))))
+				Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("BuilderInfo %q not found in namespace %q", builderName, rootNamespace))))
+			})
+		})
+
+		When("the build reconciler with the configured BuilderName is not ready", func() {
```

Comment:
Should we add a when case for the default error message?

Reply:
I've added a new test to cover this.